api/src/services/email.service.ts

```diff
@@ -143,46 +146,17 @@ export class EmailService {
     );
   }
 
-  private async send(
-    to: string | string[],
-    from: string,
-    subject: string,
-    body: string,
-    retry = 3,
-    attachment?: EmailAttachmentData,
-  ) {
-    const isMultipleRecipients = Array.isArray(to);
-    const emailParams: MailDataRequired = {
-      to,
-      from,
-      subject,
-      html: body,
-    };
-    if (attachment) {
-      emailParams.attachments = [
-        {
-          content: Buffer.from(attachment.data).toString('base64'),
-          filename: attachment.name,
-          type: attachment.type,
-          disposition: 'attachment',
-        },
-      ];
+  public async sendSES(mailOptions: SendMailOptions) {
+    try {
+      return await this.transporter.sendMail({
+        ...mailOptions,
+        from: 'Doorway <no-reply@housingbayarea.org>',
+      });
+    } catch (e) {
+      console.log(e);
+      console.error('Failed to send email');
+      return e;
```

Comment:
question: wondering if instead returning an error that we generate would be better? 

maybe throwing `InternalServerErrorException` ?

Reply:
updated ty!